### ISSUE TITLE
Add column `balanced_accuracy` to the results table of `morphoclass performance-table` 

### ISSUE THREAD

**Issue by FrancescoCasalegno** [BlueBrain/morphoclass#27]
## Context
- When we run `morphoclass performance-table`, we only report `accuracy` and `f1_score` with various averaging methods (`macro`, `micro`, `weighted`):
![Screenshot from 2022-04-20 15-51-45](https://user-images.githubusercontent.com/17013890/164247782-143978a1-52ca-4e9a-95c0-d8958e491d21.png)
- But since our dataset it would also be useful to also compute the [`balanaced_accuracy`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.balanced_accuracy_score.html#sklearn.metrics.balanced_accuracy_score), which is basically an `accuracy` with `weighted` averaging.

## Accuracy
- [ ] Add to the HTML reports of `morphoclass performance-table` a new column  `balanced_accuracy`.
